contracts/JBTiered721DelegateStore.sol

```diff
@@ -800,6 +817,9 @@ contract JBTiered721DelegateStore is IJBTiered721DelegateStore {
       if (_tierToAdd.lockedUntil != 0 && _tierToAdd.lockedUntil < block.timestamp)
         revert INVALID_LOCKED_UNTIL();
 
+      // Make sure the royalty rate is within the bounds.
+      if (_tierToAdd.royaltyRate > MAX_ROYALTY_RATE) revert INVALID_ROYALTY_RATE();
```

Comment:
@viraj124 this could use a quick test too plz!

Reply:
done